### DISCUSSION TITLE
Hubs access on the Oculus Quest using the Oculus Link cable or Virtual Desktop app

### DISCUSSION THREAD

**Discussion by nurse-the-code** [mozilla/hubs#2862] [General Questions & Discussion] [answered]
How do you access Hubs through either the Oculus Link cable or the Virtual Desktop app? For rooms that are more difficult to load on mobile devices, I would like to be able to use my computer's extra graphical and computing power to load the rooms on my Quest.

**Comment by emclaren** [comment, marked as answer]
Hi there, to my knowledge Hubs does not work with the Virtual Desktop app (yet, at least). I haven't tried it myself but in our discord one of our team mentioned "I haven't tried with Virtual Desktop, but my understanding is you need to purchase it AND also load a special version of it via SideQuest. Only then can you make it work wirelessly." Another team member added that "I have the special version on my Quest with the sideloaded modification and even I can't get it to work.  The browser only seems to try to direct you to a standalone VR device.  Would love to have that option tho if anyone can figure it out."

That being said, this information might be out of date, but I haven't tried this out myself. 

I have heard folks using the Oculus Link cable to successfully use Hubs with their Quest in our Discord server. I believe  to do this you would use Firefox to open it just like with other wired-in VR headsets. 

If anyone has any experience with this, or are finding themselves getting stuck in the process please let me know!

**Reply by ocularvr** [reply]
You connect the quest with the link to your desktop. Then with a browser, go to the room url. Once there you can put the headset on and you are good to go. This uses your computer GPU so overall quality is far higher than the the stand alone quest browser.
